### PR TITLE
Dropwizard upgrade and reworking tests

### PR DESCRIPTION
**Pull request by shrey-garg**

### CODE REVIEW

**Comment by r0goyal** [comment]
@shrey-garg Tests are failing. Please check

**Comment by shrey-garg** [comment]
@r0goyal should work now

**Comment by r0goyal** [comment]
@shrey-garg Create this PR w.r.t v0.3 branch. Let's keep one branch for next version release

File: foxtrot-core/src/main/java/com/flipkart/foxtrot/core/common/Action.java [hunk 1]
```diff
@@ -93,6 +96,9 @@ public ActionResponse execute() throws QueryStoreException {
         }
         logger.info("Cache miss for key: " + cacheKeyValue);
         parameter.setFilters(checkAndAddTemporalBoundary(parameter.getFilters()));
+        if (!parameterTableExists(parameter)) {
```
Comment (r0goyal) [inline foxtrot-core/src/main/java/com/flipkart/foxtrot/core/common/Action.java]: Move this to beginning of the method

**Pull request closed without merging**